lib/index.js

```diff
@@ -54,6 +54,7 @@ const {
  * Setting this to `true` is not normally recommended,
  * for it leads to use of an access token that is not audience restricted.
  * @param {string} [param0.sessionKey=session] The key identifying the session object attached to requests.
+ * @param {string} [param0.sessionAuthorizationKeySecret=undefined] Secret to control AuthorizationKey for horizontal scaling.
```

Comment:
Probably no need for undefined default, could it be just [param0.authorizationId]. Also, from a developer's point of view, they don't really (need to) know about the internal `authorizationKey` variable. So maybe, something like this:
> * @param {string} [param0.authorizationId] Identifier for the authorization state saved in the session object, so that an authorization could be shared between the helper instances and used for horizontal scaling.

Reply:
very good simplification suggestion! will change that